### ISSUE TITLE
css naming convention

### ISSUE THREAD

**Issue by rimmesbe** [ElenaCid/capstone_colmar#5]
There are many different professional naming conventions in css but they all pretty much use '-' or '_' to split words in selectors:
.feature-container-img
.banner-header
Check out the BEM convention: http://getbem.com/introduction/

https://github.com/ElenaCid/capstone_colmar/blob/master/resources/css/style.css#L146

**Comment by rsaftoiu** [comment]
I agree with this comment - personally I use the '-' to separate words in css selectors.
